### ISSUE TITLE
Monitor keywords/games

### ISSUE THREAD

**Issue by camjac251** [Instinctlol/automatic-twitch-recorder#1]
Would it be possible to have the script monitor a game's page and/or keywords in the title?

I'd like to record all streams of one game without having to manually startup multiple scripts for individual channels

**Comment by Instinctlol** [comment]
Yes this is possilbe. However, I'm currently occupied with university + work + another project I'm working on, so don't expect this anytime soon (maybe in a year or so).

But feel free to fork / PR.

Here's what you will need to do:

We need to use threading. Google "background thread python" or something equivalent. [Here is a good answer](http://stackoverflow.com/questions/7168508/background-function-in-python)
So then we have a main-thread and background-threads.
The main-thread watches a "game" (take a look [here](https://discuss.dev.twitch.tv/t/video-search-by-game-name/2659), e.g. for [Dota 2](https://api.twitch.tv/kraken/streams?game=Dota%202)) and starts new background-threads for every stream, who will handle the recording via livestreamer.
